config/device-classes/generic/vmware-esxi.yaml

```diff
@@ -0,0 +1,67 @@
+name: "vmware-esxi"
+
+config:
+  components:
+    cpu: true
+    memory: true
+    disk: true
+    server: true
+
+match:
+  logical_operator: OR
+  conditions:
+    - type: SysObjectID
+      match_mode: startsWith
+      values:
+        - ".1.3.6.1.4.1.6876.4.1"
+    - type: SysDescription
+      match_mode: regex
+      values:
+        - '^VMware ESXi'
+
+identify:
+  properties:
+    vendor:
+      - detection: constant
+        value: "VMware"
+    model:
+      - detection: SysDescription
+    serial_number:
+      - detection: snmpget
+        oid: .1.3.6.1.2.1.47.1.1.1.1.11.1
+    os_version:
+      - detection: snmpget
+        oid: .1.3.6.1.4.1.6876.1.1-7
+
+components:
+  cpu:
+    load:
+      - detection: snmpget
+        oid: ".1.3.6.1.2.1.25.3.3.1.2.1"
+        operators:
+          - type: modify
+            modify_method: multiply
+            value:
+              detection: constant
+              value: 100
```

Comment:
`hrProcessorLoad` provides a percent value (between 0 and 100)
Why multiply 100?

Reply:
Seems to be a copy & paste error.
I removed the calculation